cmake/CMakeLists.txt

```diff
@@ -1703,9 +1703,12 @@ if (WIN32 AND NOT GDK_PLATFORM)
   endif()
 endif()
 
-foreach(target_name ${ONNXRUNTIME_CMAKE_FILES})
+foreach (target_name ${ONNXRUNTIME_CMAKE_FILES})
   include(${target_name}.cmake)
-endforeach()
+  if (${target_name} STREQUAL "onnxruntime_unittests" AND onnxruntime_POST_MERGE_WASM_TESTING)
```

Comment:
It can be put in onnxruntime_unitest.cmake? I think it's better to keep this for-loop simple. Once we started to add target specific logic here, it could quickly grow large.